src/components/MySelect.tsx

```diff
@@ -0,0 +1,28 @@
+
+import { SelectChangeEvent, FormControl, InputLabel, MenuItem, Select } from "@mui/material";
+import React from "react";
+
+type SelectPropsType = {
+  label: string,
+  value: string,
+  options: string[],
+  onchange: (event:SelectChangeEvent<string>) => void
+}
+
+const MySelect = (props:SelectPropsType) => {
+  const {label, value, options, onchange} = props;
```

Comment:
```suggestion
const MySelect: React.VFC<SelectPropsType> = ({ label, value, options, onchange }) => {
```
haranishiさんの書き方でも全く問題ありません。
ただ今回は上記の書き方に統一してください！この記述の意味は以下のとおりです。
- MySelect: React.VFC<Props> -> MySelectコンポーネントはReactの関数コンポーネントですよとコンパイラに明示している。また、ジェネリックにPropsを渡すことで、propsの型を指定している。
- { label, value, options, onchange } -> propsを分割代入で取得し、それらを直接引数に渡している